### PR TITLE
docs: describe commit types

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**

Enhancement.

**Did you add tests for your changes?**
NA
**If relevant, did you update the documentation?**
Yes
**Summary**

Can be an alternative for #1340 

 **Motivation** - [for this change](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#type)

Screenshot from 2020-03-23 07-12-56

**Does this PR introduce a breaking change?**

NO

**Other information**

NA

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -75,7 +75,6 @@ In case you are suggesting a new feature, we will match your idea with our curre
 -   Bootstrap all the submodules before building for the first time
 
     ```bash
-    #yarn
     yarn bootstrap
     yarn build
     ```
```
Comment (himanshu010) [inline .github/CONTRIBUTING.md]: Duplicate of #1350

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -91,7 +90,7 @@ In case you are suggesting a new feature, we will match your idea with our curre
 -   Run CLI tests with:
 
     ```bash
-    yarn test:cli`
+    yarn test:cli
     ```
 
 -   Run tests of all packages:
```
Comment (himanshu010) [inline .github/CONTRIBUTING.md]: Duplicate of #1350

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -209,15 +208,15 @@ format that includes a **type** and a **subject**:
...
--   tests
+-   **ast** : init, migrate, etc.
+-   **break** : changes that break the behaviour of the cli.
+-   **chore** : Updating deps, docs, linting, etc.
+-   **cli** : changes related to core CLI things.
+-   **docs** : Documentation only changes
+-   **feat** : A new feature
+-   **fix** : A bug fix, typos, etc.
+-   **misc** : Other formats like tweaks and such.
+-   **tests** : Adding missing or correcting existing tests
```
Comment (anshumanv) [inline .github/CONTRIBUTING.md]:
```suggestion
-   **tests** : Adding missing or correcting existing tests.
```

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -209,15 +208,15 @@ format that includes a **type** and a **subject**:
...
--   feat
--   fix
--   misc
--   tests
+-   **ast** : init, migrate, etc.
+-   **break** : changes that break the behaviour of the cli.
+-   **chore** : Updating deps, docs, linting, etc.
+-   **cli** : changes related to core CLI things.
+-   **docs** : Documentation only changes
+-   **feat** : A new feature
```
Comment (anshumanv) [inline .github/CONTRIBUTING.md]:
```suggestion
-   **feat** : A new feature.
```

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -209,15 +208,15 @@ format that includes a **type** and a **subject**:
...
--   ast
--   break
--   chore
--   cli
--   docs
--   feat
--   fix
--   misc
--   tests
+-   **ast** : init, migrate, etc.
```
Comment (anshumanv) [inline .github/CONTRIBUTING.md]:
```suggestion
-   **ast** : Init, migrate, etc.
```

Comment (anshumanv) [reply .github/CONTRIBUTING.md]: This message doesn't give enough context, need an alternative

File: .github/CONTRIBUTING.md [hunk 6]
```diff
@@ -209,15 +208,15 @@ format that includes a **type** and a **subject**:
...
+-   **break** : changes that break the behaviour of the cli.
```
Comment (anshumanv) [inline .github/CONTRIBUTING.md]:
```suggestion
-   **break** : Changes that break the behaviour of the cli.
```

File: .github/CONTRIBUTING.md [hunk 7]
```diff
@@ -209,15 +208,15 @@ format that includes a **type** and a **subject**:
...
+-   **chore** : Updating deps, docs, linting, etc.
+-   **cli** : changes related to core CLI things.
```
Comment (anshumanv) [inline .github/CONTRIBUTING.md]:
```suggestion
-   **cli** : Changes related to core CLI things.
```

File: .github/CONTRIBUTING.md [hunk 8]
```diff
@@ -209,15 +208,15 @@ format that includes a **type** and a **subject**:
...
+-   **docs** : Documentation only changes
```
Comment (anshumanv) [inline .github/CONTRIBUTING.md]:
```suggestion
-   **docs** : Documentation only changes.
```

File: .github/CONTRIBUTING.md (hunk 5 above)
Comment (snitin315) [reply .github/CONTRIBUTING.md]: Something you have in mind??

Comment (anshumanv) [reply .github/CONTRIBUTING.md]: cc /@webpack/cli-team

File: .github/CONTRIBUTING.md [hunk 9]
```diff
@@ -285,7 +285,7 @@ It takes the input file given as parameter and uses jest to create a snapshot of
 This is still in a very raw form. We'd like to take this as close to a truly useful tool as possible.
 We will still need to
 
--   Support all kinds of webpack configuration(made using merge tools)
+-   Support all kinds of webpack configuration( made using merge tools ).
```
Comment (evilebottnawi) [inline .github/CONTRIBUTING.md]: No need spaces

**Comment by anshumanv** [comment]
Thanks!

**Pull request merged**